Marlin/temperature.cpp

```diff
@@ -476,18 +478,35 @@ inline void _temp_error(int e, const char *msg1, const char *msg2) {
 }
 
 void max_temp_error(uint8_t e) {
-  disable_all_heaters();
-  _temp_error(e, PSTR(MSG_MAXTEMP_EXTRUDER_OFF), PSTR(MSG_ERR_MAXTEMP));
+// Temp error has been reset
+  if (time_since_last_err[e] == 0) {
+    time_since_last_err[e] = millis();
+  }
+// There has been a recent error, if was more than a second ago, it is probably an error
+  else if (millis() > time_since_last_err[e] + 1000UL) {
```

Comment:
Yeah no problem, we could make it either a variable set in temperature.cpp or add it as a configuration option in Configuration.h